yarn.lock

```diff
@@ -15711,6 +15711,11 @@ typescript@3.9.6:
   resolved "https://registry.yarnpkg.com/typescript/-/typescript-3.9.6.tgz#8f3e0198a34c3ae17091b35571d3afd31999365a"
   integrity sha512-Pspx3oKAPJtjNwE92YS05HQoY7z2SFyOpHo9MqJor3BXAGNaPUs83CuVp9VISFkSjyRfiTpmKuAYGJB7S7hOxw==
 
+typescript@^3.8.3:
```

Comment:
Oh I guess `compile-common` never got updated with the recent bump PR. I'll update it now and regenerate the lock file.

And the cross-device error should not occur here as `init` only copies files over. The init source files are now being packaged with the other Truffle build files and just being copied instead of leveraging the unbox plumbing.